src/main/resources/application.yml

```diff
@@ -175,6 +175,10 @@ reports:
   csip:
     reconciliation:
       page-size: 20
+  visit-balance:
+    reconciliation:
+      page-size: 40
```

Comment:
There isn't very much data for visit balances - so added page size of 40 (like non-associations)